packages/ra-core/src/auth/index.ts

```diff
@@ -15,6 +15,7 @@ export * from './types';
 export * from './useAuthenticated';
 export * from './useCheckAuth';
 export * from './useGetIdentity';
+export * from './useHandleAuthCallback';
```

Comment:
I don't understand why is it called `useHandleAuthCallback` and the authProvider method is called `handleLoginCallback` ("Auth" / "Login")